data/migrations/v1.0.0-polygon.sql

```diff
@@ -27,6 +27,12 @@ UPDATE
     AND COALESCE(mz_landuse_min_zoom, 999) <> COALESCE(mz_calculate_min_zoom_landuse(planet_osm_polygon.*), 999);
 
 UPDATE planet_osm_polygon
+  SET mz_building_min_zoom = mz_calculate_min_zoom_buildings(planet_osm_polygon.*)
+  WHERE mz_calculate_min_zoom_buildings(planet_osm_polygon.*) IS NOT NULL;
```

Comment:
Fixed in 990e58ee9bc19e33cc758c40556ca7a71cfc14ec (by removing most of that WHERE line).